src/QMCWaveFunctions/tests/CMakeLists.txt

```diff
@@ -63,14 +63,14 @@ SET(JASTROW_SRC test_bspline_jastrow.cpp test_counting_jastrow.cpp test_polynomi
 SET(DETERMINANT_SRC FakeSPO.cpp test_DiracDeterminantBatched.cpp test_dirac_det.cpp
                     test_multi_dirac_determinant.cpp test_dirac_matrix.cpp
                     test_multi_slater_determinant.cpp)
-
+		  
```

Comment:
white spaces.

Reply:
Yuck, Will fix, normally these show up bright red in my emacs commit interface.